added xpadneo - linux driver for xbox one wireless gamepads

joolswills opened the pull request:
Threw this together. I've seen this driver mentioned on the forums a few times. Untested.

cmitu commented:
Looks better than my initial PR.
 One thing you could take from is the module versioning, based on the repo `VERSION` [file](https://github.com/atar-axis/xpadneo/blob/master/VERSION). The file should be copied to `$md_inst` before build.

joolswills commented:
Will do. thanks. (Sorry for late reply - I meant to reply then got sidetracked).

joolswills commented:
Sorry for delay. Tweaked it to use the VERSION file. Maybe your description etc is better I will check that also.

Removed noinstclean and added nobin.

Checkout is done outside of repo and rsynced so it's safe to clean md_inst. But nobin is probably a good flag even though I don't build driver bins. Other drivers should probably have this too.
